chain-signatures/node/src/storage/mod.rs

```diff
@@ -52,6 +52,33 @@ impl Options {
     }
 }
 
+#[derive(Debug, thiserror::Error)]
+pub enum StoreError {
+    #[error("protocol is generating: {0}")]
+    ProtocolIsGenerating(String),
```

Comment:
Protocol is a bit misleading, we are not accessing the protocol. Maybe `Entry`?

Reply:
whoops, I actually never use this error type. Was leftover from what I originally envisioned with the storage objects returning this error